src/web/js-interop.md

```diff
@@ -22,3 +22,77 @@ For help using the `js` package, see the following:
 [js-api]: {{site.pub-api}}/js
 [sass]: {{site.pub-pkg}}/sass
 
+## Future-proof JS interop
```

Comment:
"Future-proof" is a pretty big label; and one I suspect most software projects can't really promise...

Can I suggest something like "next-generation"?

Reply:
:heavy_plus_sign: Also consider adding the word "preview" or "approximation" - it may be good to highlight this is not what the finished product looks like, but a glimpse of the semantics we'll have in the future.